packages/hop-node/src/watchers/SyncWatcher.ts

```diff
@@ -357,7 +357,22 @@ class SyncWatcher extends BaseWatcher {
     // these must come after db is done syncing,
     // and syncAvailableCredit must be last
     await Promise.all(promises)
-      .then(async () => await this.syncUnbondedTransferRootAmounts())
+      .then(async () => await this.syncBonderCredit())
+  }
+
+  async syncBonderCredit () {
+    this.bonderCreditPollerIncrementer++
+    const bonderCreditSyncInterval = 10
+    // Don't check the 0 remainder so that the bonder has a valid credit immediately on startup
+    const shouldSync = this.bonderCreditPollerIncrementer % bonderCreditSyncInterval === 1
+
+    // When not uploading to S3, only sync on certain poll intervals
+    if (!this.s3Upload && !shouldSync) {
```

Comment:
since most bonders will have `s3Upload` be `false` then will it ever get past this `return`? should be an `||` instead of `&&`?

Reply:
I believe it will get past the `return` on every 10th poll. I believe it should be `&&`.

The goal was to only sync every 10 polls. For a typical bonder, I think 9 out of 10 polls will be `(true && true)`, but I think every 10th one will be `(true && false)`, which would then sync. Do you believe that to be correct?

Reply:
sorry you are correct, I had it inverted in my head. PR looks good to merge